dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/services/DataRequestAlterationManager.kt

```diff
@@ -88,10 +88,10 @@ class DataRequestAlterationManager(
     fun patchRequestStatusFromOpenToAnsweredByDataId(dataId: String, correlationId: String) {
         val metaData = metaDataControllerApi.getDataMetaInfo(dataId)
         val dataRequestEntities = dataRequestRepository.searchDataRequestEntity(
-            DataRequestsQueryFilter(
-                dataTypeFilter = metaData.dataType.value, userIdFilter = "", requestStatus = RequestStatus.Open.name,
-                accessStatus = null, reportingPeriodFilter = metaData.reportingPeriod,
-                datalandCompanyIdFilter = metaData.companyId,
+            DataRequestsFilter(
+                dataType = setOf(metaData.dataType), userId = null, emailAddress = null,
+                requestStatus = setOf(RequestStatus.Open), accessStatus = null,
```

Comment:
Does this work with the Enum? We changed it to a string in the previous story to avoid sql casting errors in the sql query

Reply:
The enum values are not inserted directly into the SQL query. The properties with the prefix `used` and suffix `filter` are actually inserted into the SQL query. And these are either booleans, strings or lists of strings.

Reply:
ok they are transformet in the DataRequestFilter.